tests/test_task.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+import torch
+
+from graphnet.models.detector.icecube import IceCube86
+from graphnet.models.graph_builders import KNNGraphBuilder
+from graphnet.models.gnn import DynEdge
+from graphnet.models.task.reconstruction import EnergyReconstruction
+from graphnet.components.loss_functions import LogCoshLoss
+
+
+def test_transform_prediction_and_target():
```

Comment:
These seem alright, so if you have checked them with `pytest` I'm all for it! :)

Reply:
I sure have!